core/services/llo/mercurytransmitter/transmitter.go

```diff
@@ -169,36 +169,41 @@ func (mt *transmitter) Start(ctx context.Context) (err error) {
 			mt.lggr.Debugw("Loading transmit requests from database")
 		}
 
-		{
-			var startClosers []services.StartClose
-			for _, s := range mt.servers {
-				transmissions, err := s.pm.Load(ctx)
-				if err != nil {
-					return err
-				}
-				s.q.Init(transmissions)
-				// starting pm after loading from it is fine because it simply
-				// spawns some garbage collection/prune goroutines
-				startClosers = append(startClosers, s.c, s.q, s.pm)
-
-				// Number of goroutines per server will be roughly
-				// 2*nServers*TransmitConcurrency because each server has a
-				// delete queue and a transmit queue.
-				//
-				// This could potentially be reduced by implementing transmit batching,
-				// see: https://smartcontract-it.atlassian.net/browse/MERC-6635
+		wg := &sync.WaitGroup{}
+		wg.Add(1)
+
+		var startClosers []services.StartClose
+		for _, s := range mt.servers {
+			transmissions, err := s.pm.Load(ctx)
+			if err != nil {
+				return err
+			}
+			s.q.Init(transmissions)
+			// starting pm after loading from it is fine because it simply
+			// spawns some garbage collection/prune goroutines
+			startClosers = append(startClosers, s.c, s.q, s.pm)
+
+			// Number of goroutines per server will be roughly
+			// 2*nServers*TransmitConcurrency because each server has a
+			// delete queue and a transmit queue.
+			//
+			// This could potentially be reduced by implementing transmit batching,
+			// see: https://smartcontract-it.atlassian.net/browse/MERC-6635
+			go func(wg *sync.WaitGroup, mt *transmitter, s *server) {
+				wg.Wait()
```

Comment:
This line now refers to the `wg` passes as param and not to the var defined on line 172. We should rename one of those.